src/activities/drawletters/Drawletters.qml

```diff
@@ -0,0 +1,287 @@
+/* GCompris - drawletters.qml
+ *
+ * Copyright (C) 2016 Nitish Chauhan <nitish.nc18@gmail.com>
+ *
+ * Authors:
+ *   Bruno Coudoin <bruno.coudoin@gcompris.net>(GTK+ version)
+ *   Nitish Chauhan <nitish.nc18@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+import QtQuick 2.2
+import GCompris 1.0
+import "../../core"
+import QtQuick.Particles 2.0
+import "drawletters.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    property bool clickanddrawflag: true
+    property real pointImageOpacity: 1.0
+
+    onStart: focus = true
+
+    onStop: {}
+
+    pageComponent: Item {
+        id: background
+        anchors.fill: parent
+
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+
+
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property GCAudio audioVoices: activity.audioVoices
+            property GCAudio audioEffects: activity.audioEffects
+            property alias bonus: bonus
+            property alias pointImageRepeater: pointImageRepeater
+            property alias segmentsRepeater: segmentsRepeater
+            property alias imageBack: imageBack
+            property int pointIndexToClick
+
+
+                    }
+
+
+        onStart: {
```

Comment:
Globally, please remove all useless line breaks and indent the code correctly (brackets...).